src/main/views/cookies.njk

```diff
@@ -0,0 +1,71 @@
+{% extends "template.njk" %}
+{% from "./macros/common-components.njk" import goBack, submitButton, backToTopButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "moj/components/filter/macro.njk" import mojFilter %}
+
+{% block pageTitle %}
+    {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+    {{ super() }}
+{% endblock %}
+
+{% block content %}
+<h1 class="govuk-heading-l">{{ title }}</h1>
+<p class="govuk-body">{{ cookiesExplanation1 }}</p>
+<p class="govuk-body">{{ cookiesExplanation2 }}</p>
+<p class="govuk-body">{{ cookiesExplanation3 }}</p>
+
+{{ govukTable({
+  caption: tableCaption,
+  captionClasses: "govuk-table__caption--m",
+  head: [
```

Comment:
would it not be more extendable to have the table loop through the headers, then rows? that way any additional cookies just need to be added to the array, and you dont need to hard code the item in the array here